ui/src/app/features/sign-up-form/sign-up-form.component.scss

```diff
@@ -13,38 +13,39 @@
  * or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
+@import 'forms.scss';
+@import 'media.scss';
 
-@import "colors.scss";
+.sign-up-card {
+  @include form-card();
 
-.logo-img {
-  width: 100%;
-}
+  .mat-input-wrapper {
+    display: flex;
+    justify-content: space-between;
 
-.mat-card-header {
-  // workaround: mat-card-header-text can not be selected (
-  margin-left: -16px;
-}
+    @include mobile-portrait {
+      flex-direction: column;
+    }
 
-.sign-up {
-  &-full-width {
-    width: 100%;
-    margin-bottom: 10px;
-  }
-}
+    ::ng-deep .mat-form-field {
+      width: 170px !important;
 
-.alert-danger {
-  background-color: $red;
-  color: $white;
-  padding: 10px;
-  margin-bottom: 10px;
-}
+      @include mobile-portrait {
+        width: 100% !important;
+      }
+    }
+  }
 
-.mat-card-actions {
-  display: flex;
-  justify-content: space-between;
-}
+  ::ng-deep {
+    .mat-checkbox-layout {
```

Comment:
same as above - looks like this should be defined generally in the theme

Reply:
Fixed